test/lsp.py

```diff
@@ -256,7 +256,7 @@ def main(self) -> int:
 
         return min(max(self.test_counter.failed, self.assertion_counter.failed), 127)
 
-    def setup_lsp(self, lsp: JsonRpcProcess, expose_project_root=True):
+    def setup_lsp(self, lsp: JsonRpcProcess, expose_project_root: Any=True) -> None:
```

Comment:
why `Any` and not `bool` here? Or did you simply walk through all complains and added `Any` to it? Which is fine, I'd just like to know about it as we're touching this file quite regularly here. :)